Client/Frontend/Browser/Tab.swift

```diff
@@ -379,11 +379,13 @@ class Tab: NSObject {
             return nil
         }
 
+        guard let title = tab.title, !title.isEmpty else { return nil }
+
         if let displayURL = tab.url?.displayURL, RemoteTab.shouldIncludeURL(displayURL) {
             let history = Array(tab.historyList.filter(RemoteTab.shouldIncludeURL).reversed())
             return RemoteTab(clientGUID: nil,
                 URL: displayURL,
-                title: tab.displayTitle,
+                title: title,
```

Comment:
What about the case when tab title isn't available? 

We should revert to showing the url instead.

Reply:
That's actually what we want to prevent. Sure a title may not be available right away when loading a page, but when it finishes loading the observer will fill in the title for sure.

Reply:
There can be cases where you will never get a title. I can have a webpage without a title and in that case what do we do?

Reply:
Hmm, that's something I would assume doesn't happen often. But it's something to think over - let me look over it and circle back with you on soon.

Reply:
Sounds good, few places I can feel like this will be very common.

Areas with bad internet connection as in if someone is taking a train / subway then in a tunnel the internet connection will take forever for a page title to load or fail to load it all together. And another case is with some website pages which don't have titles like pdf links, image links, etc.

Reply:
Hi, fly-by comment from a desktop engineer working on Firefox View :) Maybe it be worth aligning with someone on the sync team to see how they handle cases where there isn't a title for desktop?